src/flow/store/event/eventReducer.js

```diff
@@ -227,6 +231,23 @@ export default function reduce(state = initialState, action = {}) {
         }
       }
       break
+    case VERIFY_ADD_CALENDAR:
+      newState = {
+        ...state,
+        verifyAddCalendarData: payload,
+      }
+      if (
+        payload.calendar &&
+        payload.calendar.data &&
+        payload.calendar.data.src
+      ) {
+        newState['showSettingsAddCalendarUrl'] = payload.calendar.data.src
+      }
+
+      if (payload.showSettingsAddCalendarUrl != null) {
```

Comment:
This is confusing. Why clearing the field while it was added in the payload.

Reply:
Because in the end of addCalendar, I clar the verification fields. I changed this property name to better understand.